ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py

```diff
@@ -128,6 +128,44 @@ def test_tensorflow_ApproximateEqual(  # NOQA
         tolerance=tol,
     )
 
+    
+@st.composite
+def df(draw, data_format):
+    data_format = draw(data_format)
+    return data_format
+
+
+# AvgPool
+@handle_frontend_test(
+    fn_tree="tensorflow.raw_ops.AvgPool",
+    data_format=df(data_format=st.sampled_from(["NHWC"])),
```

Comment:
Supported values for `data_format` are "NHWC" and "NCHW". Also it's redundant to use a composite function here. You could do `data_format=st.sampled_from(["NHWC", "NCHW"]),`.